#!/usr/bin/env python3
PACKAGE = "publish_path"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("distance_threshold", double_t, 0, "Two points distance_threshold", 0.1, 0.0, 100)
gen.add("num_limit", bool_t, 0, "Num limit", True)
gen.add("time_limit", bool_t, 0, "Time limit", False)
gen.add("num_max_size", int_t, 0, "Num_max_size", 100000, 1, 100000000)
gen.add("time_max_limit", double_t, 0, "Time_max_limit", 10000, 0.0, 100000000)

exit(gen.generate(PACKAGE, "publish_path", "Reconfig"))